### PR TITLE
license check

### PR DESCRIPTION
**Pull request by jamezp**
This PR somewhat undoes https://issues.redhat.com/browse/WFLY-13232. This is done because of a downstream license check. I'm not entirely sure how it works, but without these changes that test does not pass.

### CODE REVIEW

**Comment by bstansberry** [comment]
@jamezp @dosoudil A change of this magnitude should be backed by a WFLY explaining what the main bits are about. (I don't mean simple things like fixing alphabetical order.) Otherwise people trying to investigate code history will have no clue what happened.

@darranl should agree to the JBoss Negotiation change. I believe it's his project and he recently did https://issues.redhat.com/browse/WFLY-13232 so there's a discrepancy of some sort.

**Comment by jamezp** [comment]
For the reasons above I'm going to place a hold on this for now. This is definitely not required.

File: servlet-feature-pack/src/license/servlet-feature-pack-licenses.xml [hunk 1]
```diff
@@ -345,8 +345,8 @@
       <artifactId>jboss-negotiation-ntlm</artifactId>
       <licenses>
         <license>
-          <name>GNU Lesser General Public License v2.1 or later</name>
-          <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html</url>
+          <name>GNU Lesser General Public License v2.1 only</name>
```
Comment (darranl) [inline servlet-feature-pack/src/license/servlet-feature-pack-licenses.xml]: This change MUST NOT be applied, the project uses the "or later" form of LGPL.

File: servlet-feature-pack/src/license/servlet-feature-pack-licenses.xml [hunk 2]
```diff
@@ -334,8 +334,8 @@
       <artifactId>jboss-negotiation-extras</artifactId>
       <licenses>
         <license>
-          <name>GNU Lesser General Public License v2.1 or later</name>
-          <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html</url>
+          <name>GNU Lesser General Public License v2.1 only</name>
```
Comment (darranl) [inline servlet-feature-pack/src/license/servlet-feature-pack-licenses.xml]: This change MUST NOT be applied, the project uses the "or later" form of LGPL.

File: servlet-feature-pack/src/license/servlet-feature-pack-licenses.xml [hunk 3]
```diff
@@ -356,8 +356,8 @@
       <artifactId>jboss-negotiation-spnego</artifactId>
       <licenses>
         <license>
-          <name>GNU Lesser General Public License v2.1 or later</name>
-          <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html</url>
+          <name>GNU Lesser General Public License v2.1 only</name>
```
Comment (darranl) [inline servlet-feature-pack/src/license/servlet-feature-pack-licenses.xml]: This change MUST NOT be applied, the project uses the "or later" form of LGPL.

File: servlet-feature-pack/src/license/servlet-feature-pack-licenses.xml [hunk 4]
```diff
@@ -323,8 +323,8 @@
       <artifactId>jboss-negotiation-common</artifactId>
       <licenses>
         <license>
-          <name>GNU Lesser General Public License v2.1 or later</name>
-          <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html</url>
+          <name>GNU Lesser General Public License v2.1 only</name>
```
Comment (darranl) [inline servlet-feature-pack/src/license/servlet-feature-pack-licenses.xml]: This change MUST NOT be applied, the project uses the "or later" form of LGPL.

**Review by darranl — CHANGES_REQUESTED** [review]
This changes for JBoss Negotiation MUST NOT be applied, the project uses the "or later" form of LGPL.

**Comment by jamezp** [comment]
Given the above comments I'm just going to close this for now.

**Pull request closed without merging**